docs/Networking.md

```diff
@@ -130,4 +130,4 @@ ws.onclose = (e) => {
 };
 ```
 
-Your app can now display all sorts of data and you may soon need to organize this content into several screens. To manage the transition between these screens, you will need to learn about [navigators](docs/using-navigators.html).
```

Comment:
Can you delete the whole paragraph altogether? This was meant to be a segue to a guide that has now been deleted. Some else opened a PR for this, I'll merge whoever's makes this change first.